README.md

```diff
@@ -301,7 +331,7 @@ For example, you can use the following command (assuming `cp` contains
 the classpath) in your `.dir-locals.el`:
 
 ```el
-((nil . (eval . (setq inf-clojure-boot-cmd (concat "lumo -d -c "
+((nil . (eval . (setq inf-clojure-generic-cmd (concat "lumo -d -c "
```

Comment:
The alignment of the second line is broken now, you probably need to change both